src/lib/Components/Consignments/Components/ImageSelection.tsx

```diff
@@ -36,92 +47,93 @@ const TakePhotoImage = (props: TakePhotoImageProps) =>
     onPress={props.onPressNewPhoto}
     style={{
       backgroundColor: "white",
+      borderWidth: 2,
+      borderColor: null,
+
       height: 158,
       width: 158,
+      margin: 4,
+      justifyContent: "center",
+      alignItems: "center",
     }}
   >
-    <Image source={{ uri: "" }} style={{ height: 158, width: 158 }} />
+    <Image source={require("../images/camera-black@2x.png")} />
   </TouchableHighlight>
 
 const ImageForURI = (props: ImagePreviewProps) =>
   <TouchableHighlight
-    onPress={() => props.onPressItem(props.image.image.uri)}
+    onPress={() => props.onPressItem(props.data.image.uri)}
     style={{
       backgroundColor: colors["gray-regular"],
       height: 158,
       width: 158,
+      margin: 4,
       borderColor: props.selected ? "white" : null,
       borderWidth: 2,
     }}
   >
-    <Image source={{ uri: props.image.image.uri }} style={{ height: 158, width: 158 }} />
+    <Image source={{ uri: props.data.image.uri }} style={{ height: 154, width: 154 }} />
   </TouchableHighlight>
 
-interface ImageViewCoupletProps {
-  first: ImageData
-  second: ImageData
-  firstSelected: boolean
-  secondSelected: boolean
-  onPressItem: (uri: string) => void
-  onPressNewPhoto: () => void
-}
-
-const ImageViewCouplet = (props: ImageViewCoupletProps) =>
-  <View style={{ flexDirection: "row" }}>
-    {props.first
-      ? <ImageForURI image={props.first} onPressItem={props.onPressItem} selected={props.firstSelected} />
-      : <TakePhotoImage onPressNewPhoto={props.onPressNewPhoto} />}
-    {props.second &&
-      <ImageForURI image={props.second} onPressItem={props.onPressItem} selected={props.secondSelected} />}
-  </View>
-
```

Comment:
This used to only support a grid with 2 columns, now it can be as many wide as can fit.